packages/belos/test/GCRODR/test_gcrodr_complex_hb.cpp

```diff
@@ -157,12 +157,12 @@ int main(int argc, char *argv[]) {
     // *****************(can be user specified)******************
     int maxits = dim/blocksize; // maximum number of iterations to run
     int numBlocks = 100;
-    int numRecycledBlocks = 80;
+    int numRecycledBlocks = 20;
```

Comment:
This is a non-backward compatible change, no?

Reply:
@bartlettroscoe this is a test.